File: JAVA/2710. Remove Trailing Zeros From a String.java
class Solution {
    public String removeTrailingZeros(String n) {
        int i,k=0;
        for(i=n.length()-1;i>=0;i--){
            if(n.charAt(i)=='0')
            k++;
            else
            break;
        }
        return (n.substring(0,n.length()-k));
    }
}

File: C++/2710. Remove Trailing Zeros From a String.cpp
class Solution {
public:
    string removeTrailingZeros(string n) {
        int i,k=0;
        for(i=n.size()-1;i>=0;i--){
            if(n[i]=='0')
            k++;
            else
            break;
        }
        return(n.substr(0,n.size()-k));
    }
};